feature/office/src/main/java/com/iteneum/office/presentation/viewmodel/OfficeViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package com.iteneum.office.presentation.viewmodel
+
+import android.util.Log
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+/**
+ * This class is the viewModel corresponding to the Office screen
+ * @author Yaritza Moreno
+ */
+@HiltViewModel
+class OfficeViewModel @Inject constructor(): ViewModel() {
+
+    //I'm just added the context parameter to test the viewModel
+    //It can be removed in the future
+    fun onClick(){
```

Comment:
Please be more descriptive on what is going to be clicked on the function. For example:

1. When the user does click on the `Call` button you can name the action like: onCallClicked()
2. When the user does click ib the `Mail` button you can name the action like: onMailClicked()

Also you can add the two functions for each bottom, remove the viewModelScope and add a TODO comment like:
//TODO add the functionality for the call button 
//TODO add the functionality for the mail button